app/models/visitors_step.rb

```diff
@@ -79,9 +84,41 @@ def validate_email
     end
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def validate_visitors
-    ages = visitors.map(&:age).compact
-    visitor_constraints.validate_visitor_ages_on self, :general, ages
-    visitor_constraints.validate_visitor_number self, :general, visitors.size
+    # It's invalid if there are no visitors, but there's no need to call the API
+    if visitors.empty?
```

Comment:
What I see strange currently is that we validate the visitors as a group in public and pvb2. I don't see sending nil an error if that is something that a user can submit from the forms.